azblob/download.go

```diff
@@ -27,6 +27,8 @@ func (azp *Storer) getTags(
 	identity string,
 ) (map[string]string, error) {
 
+	azp.log.Debugf("getTags: %s", identity)
```

Comment:
Uh, by default, please don't put debug in packages that are so deep. If there is a pressing need to have this available (even on DEBUG) please add a comment justifying the need

Reply:
I was tracking a bug where azure blob store timed out - I will remove